packages/dhis2/src/Adaptor.js

```diff
@@ -220,7 +223,7 @@ export function create(path, data, params = {}) {
     }
 
     const details = `with response ${JSON.stringify(
-      response.body?.message,
+      response.body?.message || response.body,
```

Comment:
What are we expecting to see here? We don't actually want to log the output to stdout.

By `response` my first guess is http status, which we already know.

I think I'd prefer to just remove this log, unless we're certain there's a use for it?